druntime/src/core/internal/newaa.d

```diff
@@ -45,6 +45,7 @@ struct Impl
     uint used;
     uint deleted;
     TypeInfo_Struct entryTI;
+    size_t delegate(scope const void*) nothrow hashFn;
```

Comment:
Shouldn't new members be added to the end?
Probably not a concern as this is internal.

Reply:
Yeah, it doesn't matter here, because the details are unknown to the compiler, and these are all private details. I wanted to put the function next to the entryTI, as it's taking the place of a typeinfo call. But actually, it has nothing to do with the entryTI. I can move it to the bottom.

Reply:
Moved.